### PR TITLE
fix: Set account default language in account API

### LINKED ISSUE #4855: feat: Ability to Set Account Language When Creating Account Via API
Ability to Set Account Language When Creating Account Via API

### PR DESCRIPTION
**Pull request by tejaswinichile**
# Pull Request Template

## Description

Setting Account default language in the account created via API.

Fixes #4855 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Tested with the test cases

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -570,7 +570,7 @@
     t.integer "visibility", default: 0
     t.bigint "created_by_id", null: false
     t.bigint "updated_by_id", null: false
-    t.jsonb "actions", default: "{}", null: false
+    t.jsonb "actions", default: {}, null: false
```
Comment (tejaswinichile) [inline db/schema.rb:573]: Intentional, found one issue around it while testing.

Comment (sojan-official) [reply db/schema.rb:573]: @tejaswinichile  if this was committed in a previous migration. We might need to have an accompanying migration to change this? 

Or is it that the wrong schema file was created?

**Comment by pranavrajs** [comment]
@tejaswinichile Please add this locale option to `platform/api/v1/accounts_controller` as well.

**Comment by tejaswinichile** [comment]
@pranavrajs @sojan-official comments have been fixed please have a look.

**Review by sojan-official — APPROVED** [review]
LGTM

**Pull request merged**